anypoint-mq/3.x/modules/ROOT/pages/anypoint-mq-connector-reference.adoc

```diff
@@ -509,20 +509,64 @@ Anypoint MQ Subscriber Message Source retrieves messages from the given destinat
 | Message Id a| String | ID of the message that was published |  | x
 |===
 
+[[prefetch]]
+=== Prefetch
+
+[%header%autowidth.spread]
+|===
+| Field | Type | Description | Default Value | Required
+| Max Local Messages a| Number | Maximum number of messages to be prefetched from the service and kept locally before being dispatched to the flow | 30 |
```

Comment:
"before messages are dispatched "

(removes passive voice)

Reply:
Revised to remove all passive voice!